aleph/views/bookmarks_api.py

```diff
@@ -0,0 +1,51 @@
+import logging
+from flask import Blueprint, request
+from aleph.views.util import jsonify, require
+
+
+log = logging.getLogger(__name__)
+blueprint = Blueprint("bookmarks_api", __name__)
+
+
+@blueprint.route("/api/2/bookmarks", methods=["POST"])
+def create():
+    """Creates a new bookmark for the user. This is currently a stub endpoint,
+    as the current version of the bookmarks features stores bookmarks client side.
+    ---
+    post:
+      summary: Create bookmark
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                type: object
+          description: OK
+      tags:
+        - Bookmarks
+    """
+    require(request.authz.logged_in)
+    log.info("User [%s]: Created bookmark", request.authz.role.id)
+    return jsonify({}, 200)
```

Comment:
Minor thing, but we could consider return a `201` here. Likely we also want to return the `bookmark_id`. And perhaps the URL to that bookmark (and/or set the `Location` header to it).

(not sure how (in)consistent that is with other endpoints)

Reply:
Thanks Chris!

> Minor thing, but we could consider return a 201 here.

Fair enough -- I think most of the other endpoints (e.g. entity creation) return `200`, but I agree that the better option here would be `201`.

> but it seems important to get the API right now.

I think we can change the API later on. To be honest I’m not really confident enough to commit on a stable API at the moment, especially given that the feature may change significantly or may even be removed completely based on user feedback. It’s an experimental feature though and has to be enabled explicitly, so maybe that’s ok?

Reply:
Totally fine. In that context feel free to skip the `201` change as well and check it in as it is, from my point of view 👍